src/System.Net.Requests/tests/HttpRequestResponseSerializationTest.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization.Formatters.Tests;
+using Xunit;
+
+namespace System.Net.HttpRequest.Tests
+{
+    public partial class HttpRequestResponseSerialization
+    {
+        [OuterLoop]
+        [Theory]
+        public void SerializeDeserialize_Roundtrips(Uri remoteServer)
+        {
+            HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
+            request.KeepAlive = true;
+            request.CookieContainer = new CookieContainer(10);
+            HttpWebRequest clone = BinaryFormatterHelpers.Clone(request);
+            Assert.NotSame(request, clone);
+            Assert.Equal(request, clone);
```

Comment:
Actually HttpClientHandler does NOT have a Timeout property.

The property is part of HttpClient:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/ref/System.Net.Http.cs#L44

But it can be used to support this HttpWebRequest.Timeout scenario.

Since HttpWebRequest creates a new HttpClient for each request, it can use HttpClient's Timeout property for support the concept.